docs/coordinates.rst

```diff
@@ -0,0 +1,219 @@
+===========
+Coordinates
+===========
+
+Coordinate systems in astronomy can be complicated, AstroPhot is no
+different. Here we explain how coordinate systems are handled to help
+you avoid possible pitfalls.
+
+Basics
+------
+
+There are three main coordinate systems to think about.
+
+#. ``world`` coordinates are the classic (RA, DEC) that many
+   astronomical sources are represented in. These should always be
+   used in degree units as far as AstroPhot is concerned.
+#. ``plane`` coordinates are the tangent plane on which AstroPhot
+   performs it's calculations. Working on a plane makes everything
+   linear and does not introduce a noticible effect for small enough
+   images. In the tangent plane everything should be represented in
+   arcsecond units.
+#. ``pixel`` coordinates are specific to each image, they start at
+   (0,0) in the center of the [0,0] indexed pixel. These are
+   effectively unitless, a step of 1 in pixel coordinates is the same
+   as changing an index by 1. Though image array indexing is flipped
+   so pixel coordinate (1,0) represents the center of the index [0,1]
```

Comment:
That's a good question. Every image I've encountered has this kind of indexing. I'm not sure how to detect if this is not the case. It's also fairly easy to fix on the user side by just taking the transpose before passing the image to AstroPhot. 

I'm happy to add more explanation here at least so that users know exactly what is happening under the hood.

Reply:
I don't think there's anything that the user should ever have to do about this.  Any "flipping" from detectors being oriented in different ways is encoded in the WCS.

The question I was trying to get at is the indexing you're referring to here sounds like a question of row-major or column-major order.

Reply:
yes row/column major order is probably what I meant (based on wikipedia). I don't think users will need to worry too much about this but sometimes one has to convert things around. For example GALFIT works in pixel coordinates so someone may want to compare GALFIT results and AstroPhot results and so they'll need to be really clear about what pixel goes where. Not sure if this information will help or hurt more for most users. It's something I'm happy to revisit, but I think I'll keep it for now.